app/views/restaurants/show.html.erb

```diff
@@ -43,9 +43,25 @@
 </p>
 
 <%= link_to 'Back', restaurants_path %>
+
 <br />
 <% if current_user.try(:admin?) || current_user.id == @restaurant.user_id %>
   <%= link_to 'Edit restaurant', edit_restaurant_path(@restaurant) %>
   <br />
   <%= link_to 'Delete restaurant', restaurant_path(@restaurant), method: :delete, data: { confirm: 'Are you sure?' } %>
 <% end %>
+
+<% if user_signed_in? %>
+  <%= link_to 'Write Review', new_restaurant_review_path(@restaurant) %>
+<% end %>
+
+<% @restaurant.reviews.each do |r| %>
```

Comment:
there is a scalability concern here. What happens if a restaurant has thousands of reviews? This list should be paginated.